db/migrate/20151119221200_create_doorkeeper_tables.rb

```diff
@@ -1,46 +1,38 @@
 class CreateDoorkeeperTables < ActiveRecord::Migration
   def change
     create_table :oauth_applications do |t|
-      t.string  :name,         null: false
-      t.string  :uid,          null: false
-      t.string  :secret,       null: false
-      t.text    :redirect_uri, null: false
-      t.string  :scopes,       null: false, default: ''
+      t.string :name,         null: false
+      t.string :uid,          null: false
+      t.string :secret,       null: false
+      t.text :redirect_uri, null: false
+      t.string :scopes,       null: false, default: ''
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.